client.go

```diff
@@ -27,6 +27,36 @@ import (
 	"github.com/gopcua/opcua/uasc"
 )
 
+// FindServers returns the servers known to a server or discovery server.
+func FindServers(ctx context.Context, endpoint string, opts ...Option) ([]*ua.ApplicationDescription, error) {
+	opts = append(opts, AutoReconnect(false))
+	c := NewClient(endpoint, opts...)
```

Comment:
Can you rebase this PR on top of #674 so that we can update this line and the one in `FindServersOnNetwork` ?